cmd/tf_operator/main.go

```diff
@@ -17,33 +19,37 @@ import (
 
 	log "github.com/golang/glog"
 
+	"io/ioutil"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/pkg/api/v1"
 	"k8s.io/client-go/tools/record"
+	"mlkube.io/pkg/spec"
 )
 
 var (
 	namespace  string
 	name       string
 	gcInterval time.Duration
 
-	chaosLevel int
-
-	printVersion bool
+	chaosLevel           int
+	controllerConfigFile string
+	printVersion         bool
 )
 
 var (
-	leaseDuration = 15 * time.Second
-	renewDuration = 5 * time.Second
-	retryPeriod   = 3 * time.Second
-	tfJobClient   *k8sutil.TfJobRestClient
+	controllerConfig = spec.ControllerConfig{}
+	leaseDuration    = 15 * time.Second
+	renewDuration    = 5 * time.Second
+	retryPeriod      = 3 * time.Second
+	tfJobClient      *k8sutil.TfJobRestClient
 )
 
 func init() {
 	// chaos level will be removed once we have a formal tool to inject failures.
 	flag.IntVar(&chaosLevel, "chaos-level", -1, "DO NOT USE IN PRODUCTION - level of chaos injected into the etcd clusters created by the operator.")
 	flag.BoolVar(&printVersion, "version", false, "Show version and quit")
 	flag.DurationVar(&gcInterval, "gc-interval", 10*time.Minute, "GC interval")
+	flag.StringVar(&controllerConfigFile, "controller_config_file", "", "Path to file containing the controller config.")
```

Comment:
replace `controller_config_file` by `controller-config-file` to stick with convention?